packages/stores/src/queries-external/store.ts

```diff
@@ -98,4 +112,21 @@ export class QueriesExternalStore {
         indexerDataBaseUrl
       );
   }
+  protected get bech32Address() {
+    return this.accountStore?.getWallet(this.chainId)?.address ?? "";
+  }
+  setAccountStore(
+    accountStore: AccountStore<[OsmosisAccount, CosmosAccount, CosmwasmAccount]>
+  ) {
+    this.accountStore = accountStore;
+
+    if (this.accountStore) {
+      this.queryRewardsInner = new ObservableQueryRewardsInner(
+        this.kvStore,
+        this.chainId,
+        this.chainGetter,
+        this.bech32Address
+      );
+    }
+  }
```

Comment:
As stated above, the queryRewards store can be accessed from `queriesStore.get(chainId).cosmos` (something like that).

Remember how we queried inflation?